README.md

```diff
@@ -215,6 +215,36 @@ The possible values for the annotation can be
 * `"my-instrumentation"` - name of `Instrumentation` CR instance.
 * `"false"` - do not inject
 
+#### Multi-container pods
+
+If nothing else is specified, instrumentation is performed on the first container available in the pod spec.
+In some cases (for example in the case of the injection of an Istio sidecar) it becomes necessary to specify on which container(s) this injection must be performed.
+
+For this, it is possible to fine-tune the pod(s) on which the injection will be carried out.
+
+For this, we will use the "instrumentation.opentelemetry.io/container-names" annotation for which we will indicate one or more pod names (.spec.containers.name) on which the injection must be made:
+
+```yaml
+  template:
+    metadata:
+      labels:
+        app: my-pod-with-sidecar
+      annotations:
+        sidecar.opentelemetry.io/inject: "true"
```

Comment:
remove the sidecar annotaiton - keep this example focused only on a single feature